ai2thor/controller.py

```diff
@@ -503,7 +504,11 @@ def scenes_in_build(self):
 
         return self._scenes_in_build
 
-    def reset(self, scene='FloorPlan_Train1_1'):
+    def reset(self, scene=''):
```

Comment:
Can you make the default for scene None instead of an empty string?  With the current code you could do this:

```
controller.reset(None)
```
and you would get an error during concatenation

If you make the default None and someone did do something like pass in an empty string, they would get an error that the scene does not exist.

Reply:
Done